src/nodes.coffee

```diff
@@ -1104,47 +1115,77 @@ exports.Obj = class Obj extends Base
 
   children: ['properties']
 
+  isAssignable: ->
+    @lhs = true
```

Comment:
Yes, precisely. Unfortunately it's the only reliable way I could think to do it without:

- Traversing the tree in `Assign` (e.g. `@variable.traverseChildren`). This is a bit awkward and probably bad for performance.
- Putting a property on `o` (e.g. `o.assignment = true`). This would be pretty seamless, but the `o` API can be hard to understand. This is probably the best alternative though.